src/Arriba/Arriba/Server/Authorization/ArribaIdentityNormalizer.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Linq;
+using System.Security.Claims;
+using System.Security.Principal;
+using Arriba.ParametersCheckers;
+
+namespace Arriba.Server.Authorization
+{
+    public class ArribaIdentityNormalizer : IArribaIdentityNormalizer
+    {
+        private const string ClaimTypeName = "name";
+
+        public ClaimsPrincipal NormalizeIdentity(ClaimsPrincipal user, bool throwExceptionWhenNameNotFound = true)
+        {
+            user.ThrowIfNull(nameof(user));
+            ParamChecker.ThrowIfNull(user.Identity, nameof(user.Identity));
+            user.Identity.AuthenticationType.ThrowIfNullOrWhiteSpaced(nameof(user.Identity.AuthenticationType));
+
+            var nameClaim = user.Claims.Where(c => c.Type == ClaimTypeName).FirstOrDefault();
```

Comment:
Should use String.Equals, likely with case insensitivity. Do does it matter if this is case sensitive?

Reply:
I Identity.HasClaims is case sensitive and the database security too.